### PR TITLE
add EMEA IP addresses

### PR DESCRIPTION
**Pull request by ernestoongaro**
## Description & motivation
For EMEA MT launch
https://dbt-labs.slack.com/archives/C039T8U9PMG/p1660679891261469?thread_ts=1660302363.792429&cid=C039T8U9PMG

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 1]
```diff
@@ -9,13 +9,16 @@ dbt Cloud will always connect to your warehouse from the following IP addresses.
...
+| Region/Deployment | IP Addresses |
+| ------ | ----------- |
+| US (cloud.getdbt.com) |  52.45.144.63 <br /> 54.81.134.249 <br /> 52.22.161.231 |
+| EMEA (emea.dbt.com) |  3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 |
+| Virtual Private dbt | Ask [Support](https://docs.getdbt.com/guides/legacy/getting-help#dbt-cloud-support) for your  IPs | 
 
 <Changelog>
 
 - `54.81.134.249` and `52.22.161.231` were added in November, 2020
+- EMEA was added in September, 2022
```
Comment (dataders) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: @ernestoongaro is it not available already or only in beta or something? I get that you'd like to just make only one PR here, but it feels weird to me to use the past tense for the future. @runleonarun @nghi-ly  do we have date-gating block or is there any best practice that you'd recommend?

**Review by dataders — APPROVED** [review]
obviously this change is needed, I only have one nitpick about the "changelog" section...

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 2]
```diff
@@ -9,13 +9,16 @@ dbt Cloud will always connect to your warehouse from the following IP addresses.
...
-- 52.22.161.231
+| Region/Deployment | IP Addresses |
+| ------ | ----------- |
+| US (cloud.getdbt.com) |  52.45.144.63 <br /> 54.81.134.249 <br /> 52.22.161.231 |
+| EMEA (emea.dbt.com) |  3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 |
+| Virtual Private dbt | Ask [Support](https://docs.getdbt.com/guides/legacy/getting-help#dbt-cloud-support) for your  IPs | 
 
 <Changelog>
 
 - `54.81.134.249` and `52.22.161.231` were added in November, 2020
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
- November 2020 &mdash; add the IPs `54.81.134.249` and `52.22.161.231` 
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md (hunk 1 above)
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
- September 2022 &mdash; Add EMEA IPs
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: @dataders , @ernestoongaro : i'm a fan of using present tense. I use past and future tense very sparingly in techdocs. 

left a couple suggs. feel free to accept them if 👍 

also, what do you think about making the changelog list be most recent first? maybe that's not our convention though.

**Pull request merged**